cluster/cluster.go

```diff
@@ -231,8 +230,20 @@ func (cluster *Cluster) GetContentList() []int {
 	return cluster.ContentIDs
 }
 
+func (cluster *Cluster) GetDbidForContent(contentID int) int {
+	return cluster.Segments[contentID].DbID
```

Comment:
Does this make sense? We should have two DbIDs per content ID so you shouldn't be able to get a single dbid as a result for one content id.